DataFormats/L1Trigger/interface/BXVector.icc

```diff
@@ -1,6 +1,8 @@
+#include <cassert>
+
+// FIXME: these 3 lines are required by other packages
```

Comment:
Looks like the 3 lines below are not needed in this file, but they are needed to make other packages compile. Fixing that is beyond the scope of this PR.